packages/central-server/src/social/getLeaderboard.js

```diff
@@ -33,19 +33,18 @@ const INTERNAL_EMAIL = '@beyondessential.com.au';
  */
 export const getLeaderboard = async (models, rowCount = 10) => {
   return models.database.executeSql(
-    `
-        SELECT r.user_id, user_account.first_name, user_account.last_name, r.coconuts, r.pigs
-        FROM (
-          SELECT user_id, COUNT(*) as coconuts, FLOOR(COUNT(*) / 100) as pigs
-          FROM survey_response
-          GROUP BY user_id
-        ) r
-        JOIN user_account on user_account.id = r.user_id
-        WHERE email NOT LIKE '%${INTERNAL_EMAIL}'
-        AND email NOT IN (${EXCLUDED_USERS.join(',')})
-        ORDER BY coconuts DESC
-        LIMIT ?;
-      `,
+    ` SELECT r.user_id, user_account.first_name, user_account.last_name, r.coconuts, r.pigs
+      FROM (
+        SELECT user_id, COUNT(*) as coconuts, FLOOR(COUNT(*) / 100) as pigs
+        FROM survey_response
+        GROUP BY user_id
+      ) r
+      JOIN user_account on user_account.id = r.user_id
+      WHERE ${INTERNAL_EMAIL.map(email => `email NOT LIKE '%${email}'`).join(' AND ')}
+      AND email NOT IN (${EXCLUDED_USERS.join(',')})
+      ORDER BY coconuts DESC
+      LIMIT ?;
```

Comment:
Great call to try and keep things DRY here!

What I might recommend instead of moving this to a route in the central-server, is instead we move this function to the `SurveyResponseModel` in `@tupaia/database`. Since it's really a leader board of survey responses (though I think an argument could be made for the `UserModel` as well). I think that allows us to keep the api of central-server a little cleaner :+1: